### DISCUSSION TITLE
Lost config and connection; Can't seem to reconnect

### DISCUSSION THREAD

**Discussion by agw421** [node-alarm-dot-com/homebridge-node-alarm-dot-com#121] [Q&A]
Hey all,

Thx ahead of time for reading and your help! I'm struggling to get thru the critical login portion of this setup. I've done it before with no issues that I can remember, but this time around - I'm getting an error that I can't seem to avoid. Then I immediately get locked out of my ADT account and have to reset the password and the whole nine. I'm hoping I can figure out what this error means and get rid of the issue.

From what I see, I'm definitely grabbing the right mfaCookie string and I have useMFA set to true.  I can also guarantee my login is correct. Any other issues that come to mind to fix this?

Would be greatly appreciative to have my remote garage and front door controls back online!

`{
            "name": "ADT",
            "username": "***",
            "password": "***",
            "useMFA": true,
            "mfaCookie": "A3B58C343BC3A09D8F791869596106B0A2537A8C6FBEC0BD01940C91EAEAD14A",
            "logLevel": 3,
            "armingModes": {
                "away": {
                    "noEntryDelay": false,
                    "silentArming": false,
                    "nightArming": false
                },
                "night": {
                    "noEntryDelay": false,
                    "silentArming": false,
                    "nightArming": false
                },
                "stay": {
                    "noEntryDelay": false,
                    "silentArming": false,
                    "nightArming": false
                }
            },
            "_bridge": {
                "username": "***",
                "port": 55083
            },
            "platform": "Alarmdotcom"
        },`
        
        
        And the error I'm receiving:
      `[11/10/2023, 5:10:35 PM] [ADT] loginSession Error: POST https://www.alarm.com/web/Default.aspx failed: No afg cookie: ASP.NET_SessionId=2br3pnyclxoqt3wzzzf5vez5; twoFactorAuthenticationId=A3B58C343BC3A09D8F791869596106B0A2537A8C6FBEC0BD01940C91EAEAD14A; cookieTest=1; IsFromNewSite=1; fromASP=; IsFromSeamlessLogin=; login=; ST=`

**Comment by chase9** [comment]
Looks like this was closed, did you get it figured out?

**Comment by agw421** [comment]
As soon as i posted it, something clicked and it’s online! Now i’m just
struggling with silly Homekit related things but this plugin is officially
working! thanks for replying so fast.
